docker-compose.yml

```diff
@@ -161,21 +161,23 @@ services:
       KAFKA_LDAP_JAVA_NAMING_FACTORY_INITIAL: com.sun.jndi.ldap.LdapCtxFactory
       KAFKA_LDAP_COM_SUN_JNDI_LDAP_READ_TIMEOUT: 3000
       KAFKA_LDAP_JAVA_NAMING_PROVIDER_URL: ldap://openldap:389
-      # How to authenticate to LDAP
+      # Authenticate to LDAP
       KAFKA_LDAP_JAVA_NAMING_SECURITY_PRINCIPAL: cn=admin,dc=confluentdemo,dc=io
       KAFKA_LDAP_JAVA_NAMING_SECURITY_CREDENTIALS: admin
       KAFKA_LDAP_JAVA_NAMING_SECURITY_AUTHENTICATION: simple
-      # How to locate users and groups
-      KAFKA_LDAP_SEARCH_MODE: USERS
+      # Locate LDAP users and groups
+      KAFKA_LDAP_SEARCH_MODE: GROUPS
+      KAFKA_LDAP_GROUP_SEARCH_BASE: ou=groups,dc=confluentdemo,dc=io
+      KAFKA_LDAP_GROUP_NAME_ATTRIBUTE: cn
+      KAFKA_LDAP_GROUP_OBJECT_CLASS: group
+      KAFKA_LDAP_GROUP_MEMBER_ATTRIBUTE: memberUid
+      KAFKA_LDAP_GROUP_OBJECT_CLASS: posixGroup
+      KAFKA_LDAP_GROUP_MEMBER_ATTRIBUTE_PATTERN: cn=(.*),ou=users,dc=confluentdemo,dc=io
       KAFKA_LDAP_USER_SEARCH_BASE: ou=users,dc=confluentdemo,dc=io
-      KAFKA_LDAP_GROUP_SEARCH_BASE: ou=users,dc=confluentdemo,dc=io
       KAFKA_LDAP_USER_NAME_ATTRIBUTE: uid
       KAFKA_LDAP_USER_OBJECT_CLASS: inetOrgPerson
       KAFKA_LDAP_USER_MEMBEROF_ATTRIBUTE: ou
```

Comment:
I do not see where ou defined?

Reply:
@jeffhuang26 the LDIFs are defined here: https://github.com/confluentinc/cp-demo/tree/5.4.1-post/scripts/security/ldap_users

I'm not familiar with what it _should_ be, but there is https://github.com/confluentinc/cp-demo/blob/5.4.1-post/scripts/security/ldap_users/18_app.ldif#L1 .  Is that sufficient?